WordPress/src/main/res/layout/reader_blog_info_view.xml

```diff
@@ -78,6 +79,37 @@
             app:wpFontFamily="merriweather"
             tools:text="text_description" />
 
-    </FrameLayout>
+        <LinearLayout
```

Comment:
Unfortunately, nesting the followers count inside the `layout_blog_description` hides it when the blog has no description. Calypso seems to show the followers count even in that case.